flink/v1.15/build.gradle

```diff
@@ -146,7 +147,7 @@ project(":iceberg-flink:iceberg-flink-runtime-${flinkMajorVersion}") {
     }
 
     // for dropwizard histogram metrics implementation
-    implementation "org.apache.flink:flink-metrics-dropwizard:${flinkMajorVersion}"
+    implementation "org.apache.flink:flink-metrics-dropwizard:${flinkVersion}"
```

Comment:
thx for fixing this bug

Reply:
@rdblue I believe this is the main fix. previously, it is using the wrong version number of `1.15`. should be the full version of `1.15.0`.

Reply:
the main fix is actually moving `exclude group: 'org.apache.flink'` out of the entire `implementation` configuration so that `org.apache.flink:flink-metrics-dropwizard` can make it into the runtime jar. Once I did that, I noticed that it's trying to pick up version `1.15` (which doesn't exist) rather than `1.15.0`